android/Omnivore/app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,7 @@
 <resources>
     <string name="app_name">Omnivore</string>
-    <string name="gcp_id">590528454690-mpmbvcb4c3ifmnojmrultt37eo8f64at.apps.googleusercontent.com</string>
-    <string name="welcome_title">Everything you read.\nSafe, organized, and\neasy to share.</string>
+    <string name="gcp_id">590528454690-g535h6sd708andjrvrmqgaj1sergtqi9.apps.googleusercontent.com</string>
```

Comment:
@jacksonh Did this one work for you?

Reply:
oh no, let me revert that. Was just trying it out.

Reply:
Thanks for catching, that would have been a pain to debug.